.env.integration

```diff
@@ -33,6 +33,9 @@ CELOTOOL_DOCKER_IMAGE_TAG="celotool-dfdc3e8b26e98aa294b27e2b5621c184488a10db"
 TRANSACTION_METRICS_EXPORTER_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/celo-monorepo"
 TRANSACTION_METRICS_EXPORTER_DOCKER_IMAGE_TAG="transaction-metrics-exporter-d3d165a7db548d175cd703c86c20c1657c04368d"
 
+ATTESTATION_SERVICE_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/attestation-service"
+ATTESTATION_SERVICE_DOCKER_IMAGE_TAG="nam1"
```

Comment:
Should this be changed?

Reply:
Yup will do once Cloud Build will build the official image